skorch/tests/test_regressor.py

```diff
@@ -36,6 +36,23 @@ def net(self, net_cls, module_cls):
             lr=0.1,
         )
 
+    @pytest.fixture(scope='module')
+    def multi_target_data(self, multi_target_regression_data):
+        return multi_target_regression_data
+
+    @pytest.fixture(scope='module')
+    def multi_target_module_cls(self):
+        from skorch.toy import make_regressor
+        return make_regressor(output_units=3, dropout=0.5)
+
+    @pytest.fixture(scope='module')
+    def multi_target_net(self, net_cls, multi_target_module_cls):
+        return net_cls(
+            multi_target_module_cls,
+            max_epochs=20,
```

Comment:
Please only use 1 epoch. For the test, it doesn't matter how well the model is fit, but the test time will increase considerably with 20 epochs.

Reply:
Ok but in your ```net``` function (single output regression) you used ```max_epochs=20```. Should I fix this?